doc/source/admin/framework_dependencies.rst

```diff
@@ -171,149 +171,21 @@ create a virtualenv can be found under the `Managing dependencies manually`_ sec
 Conda
 ^^^^^
 
-`Conda`_ and `virtualenv`_ are incompatible. However, Conda provides its own environment separation functionality in the
-form of `Conda environments`_.  Starting Galaxy with Conda Python will cause Galaxy to create a ``_galaxy_YY.MM``
-environment into which Galaxy framework dependencies will be installed.
-
-To instruct Galaxy to use a different environment name, set the ``$GALAXY_CONDA_ENV`` environment variable.
-
-If you use the recommended setup of sourcing ``$CONDA_ROOT/etc/profile.d/conda.sh`` as of Conda 4.4, Conda Python is not
-automatically added to your ``$PATH``, so Galaxy will not use Conda for its framework dependencies unless you start it
-with an environment activated that has Conda Python installed. This is because Galaxy defaults to creating a virtualenv
-unless the ``python`` on ``$PATH`` is Conda Python.
+`Conda`_ and `virtualenv`_ are incompatible, unless an adapted `virtualenv_` from the `conda-forge_` channel is used.
+Galaxy can create a virtualenv using the adapted virtualenv package. Once a valid ``.venv`` environment exists it will be used.
 
 .. tip::
 
-    If you would like to force Galaxy to use Conda with Conda 4.4 or later, the simplest method is:
+    If you would like to use a virtualenv created by Conda, the simplest method is:
 
         1. Ensure ``.venv`` does not exist.
-        2. Activate the base environment with ``conda activate base``
-
-    Galaxy will not install in to the base environment.
-
-.. caution::
-
-    Versions of Galaxy prior to 18.05 could install in to the base/root Conda environment. Consult the correct version
-    of the documentation for your version of Galaxy.
```

Comment:
Should we keep this warning?

Reply:
Not sure if this would clarify or make things more confusing? I could add a warning to the top that this applies to 19.01 or newer, for older version consult the documentation for your version of Galaxy ?

Reply:
Yeah, that seems reasonable.